packages/config/config/devices/0x0234/templates/logic_group_template.json

```diff
@@ -526,5 +526,77 @@
 		"$import": "../../templates/master_template.json#base_0-255_nounit",
 		"label": "Input: Timer",
 		"defaultValue": 0
+	},
+	"startup_relay": {
+		"label": "Relay Startup State",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 2,
+		"defaultValue": 0,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Off",
+				"value": 0
+			},
+			{
+				"label": "On",
+				"value": 1
+			},
+			{
+				"label": "Last registered state",
+				"value": 2
+			}
+		]
+	},
+	"watt_threshold_change": {
+		"label": "Watt Threshold: Change",
+		"unit": "watt",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 500,
+		"defaultValue": 50,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"watt_threshold_power": {
+		"label": "Watt Threshold: Power Off",
```

Comment:
Does this label make sense? What's the difference between the two? What does Power Off mean?

Reply:
You can set an upper limit of wattage consumption. 
If this number is reached lets say 2900 out of 3000 as its max outpu. it can then trigger and turn off